subjects/gatecrashers/README.md

```diff
@@ -0,0 +1,64 @@
+## Gatecrashers
+
+### Instructions
+
+Unfortunately many of your guests started to invite plus ones, that started to invite plus ones, that started... to be short, everybody is inviting everybody and the situation is rapidly going out of control.
+
+To fix this issue you will introduce a [Basic Access Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) on your server.
+
+To modify the guest list now your friends will need to make authenticated requests. You decided that only your 3 best friends (`Caleb_Squires`, `Tyrique_Dalton` and `Rahima_Young`) will be able to modify the list. You also told them the secret password `abracadabra` in order to do that.
+
+The request query will look like this `curl -u Caleb_Squires:abracadabra ...`.
+
+The server should properly handle unauthorized requests using the error code `401`.
+
+Apart for the authentication part your server's behavior should remain unchanged:
+- You will listen to port `5000` and print a message containing the port you are using.
+- For each **authorized** http `POST` request, your program should create the corresponding JSON file and store the contents of the body, and then provide the content as JSON in the HTTP response, if possible. 
```

Comment:
**SUGGESTION** To simplify the subject (and the tests) and put the focus only on the authentication part, this exercise might only ask to return the body of the request (with some additional information eventually) to avoid having a "too complex" ask for a single exercise

Reply:
as discussed it would be better to leave it as it is, since it will be strange for the student to remove a feature he already implemented in a previous exercise.
In case the exercise will be used as a stand alone one it may be made simpler, but as one in a chain that already builds up the most part of the basecode the complexity introduced is quite manageable.